SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -71,6 +71,50 @@ UnitTest {
 		}
 	}
 
+	// run a list of testItems which can be UnitTest classes or method names
+	// in the format "TestPolyPlayerPool:test_prepareChildrenToBundle"
+
+	*runTests { |testItems, postTimes = false|
+		^this.forkIfNeeded {
+			var startTime = Main.elapsedTime;
+			this.reset;
+			testItems = testItems ?? { this.allSubclasses };
+			testItems.do ({ |testItem|
+				var myStartTime = Main.elapsedTime;
+				var myTestTime;
+				if (testItem.isKindOf(Class)) {
+					// assume this is a UnitTest:
+					if (testItem.superclasses.includes(UnitTest)) {
```

Comment:
`testItem.isKindOf(UnitTest)`

Reply:
> I think the method arguments should be analogous to runTest, if the name is so similar.

well, I think they are quite similar: 
the testItems argument is a list of tests, analogous to a single TestXYZ:test_abc in runTest;
it also allows just class names (as  shorthand for all test methods in TestXYZ). 
and postTimes is a second argument. 

is the method name runTests misleading / not ideal?
would runTestItems be better?

Reply:
ok, then it's fine!

Reply:
so just one requested change.

Reply:
Hi J, your requested change does not do the same thing as the original code:
```isKindOf``` tests whether an instance comes from that class hierarchy; 
but testItems is a class, not an instance, so it needs the clumsy-looking but correct:
```
testItem.superclasses.includes(UnitTest)
```
```
// simple test with a class:
Array.superclasses.includes(Collection); // true
Array.isKindOf(Collection) // false, because isKindOf is for instances, not classes.
```

Reply:
then it is `testItem.isKindOf(Meta_UnitTest)`, no?

Reply:
yes of course, much better. will change it to that. thanks!